packages/agoric-cli/src/commands/oracle.js

```diff
@@ -175,8 +175,8 @@ export const makeOracleCommand = (logger, io = {}) => {
       console.warn(sendHint);
     });
 
-  const findOracleCap = async (instance, from, readLatestHead) => {
-    const current = await getCurrent(from, { readLatestHead });
+  const findOracleCap = async (instance, from, readPublished) => {
+    const current = await getCurrent(from, { readPublished });
```

Comment:
oops. In order to be compatible, this needed changes to all its callers.

There are callers that still supply `readLatestHead`, which requires that the `published.` prefix is explicit.

```
      const offerId = await findOracleCap(instance, opts.from, readLatestHead);
```